guides/v2.3/graphql/functional-testing.md

```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
 $registry->register('isSecureArea', false);
 ```
 
+### Fixture configs
+
+Use `@magentoConfigFixture` annotation to set a custom config value. It supports a `store` scope only.
+ 
+#### Syntax
+
+```php
+/**
+ * @magentoConfigFixture <store_code>_store <config_key> <config_value>
+ */
+```
+
+where
+- `<store_code>` - Store code. See the `store`.`code` database field value.
+- `<config_key>` - Config key. See `core_config_data`.`path` 
+- `<config_value>` - Config value. See `core_config_data`.`value`
+
+{: .bs-callout-info }
+`@magentoConfigFixture` does not require a roll-back.  
+
+#### Example usage 
+ 
+The following example sets a store-scoped value `1` for the config key `checkout/options/enable_agreements` for the `default` store in the `GetActiveAgreement()` test: 
+
+```php
+    /**
+     * @magentoConfigFixture default_store checkout/options/enable_agreements 1
+     */
+    public function testGetActiveAgreement()
+    {
+        ...
+    }
+```
+
+In the background process, `@magentoConfigFixture` performs the following action before test execution:  
```

Comment:
```suggestion
`@magentoConfigFixture` performs the following action  as a background process before test execution:  
```

Reply:
**fixed**

Reply:
@keharper , all your suggestions have been implemented.

Thank you!